package.json

```diff
@@ -19,7 +19,7 @@
     "gatsby-image": "^2.4.5",
     "gatsby-plugin-manifest": "^2.4.9",
     "gatsby-plugin-react-helmet": "^3.3.2",
-    "gatsby-plugin-sharp": "^2.6.9",
+    "gatsby-plugin-sharp": "^2.6.42",
```

Comment:
please revert all these changes, you should only change you json file. You could revert these by copying the respective files from the repository.
You can see other merged PRs for help.

Reply:
Thanks will do that. After I make these changes do I need to send another PR?

Reply:
@Abhishek-kumar09 I have reverted all these changes in the next commit